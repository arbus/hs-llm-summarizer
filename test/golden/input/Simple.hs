module Simple where

-- | A simple function that adds two numbers
add :: Int -> Int -> Int
add x y = x + y

-- | A simple data type
data Person = Person 
  { name :: String
  , age :: Int
  } deriving (Show, Eq)

-- | Create a person with given name and age
createPerson :: String -> Int -> Person
createPerson n a = Person n a