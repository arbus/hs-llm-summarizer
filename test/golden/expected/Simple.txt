-- Module: test/golden/input/Simple.hs

 | A simple function that adds two numbers

add :: Int -> Int -> Int



 | A simple data type

data Person = Person{name :: String, age :: Int}
                deriving (Show, Eq)

 | Create a person with given name and age

createPerson :: String -> Int -> Person

